README.md

```diff
@@ -12,23 +12,23 @@ A Debian-based distribution.
 
 ## Role Variables
 
-| Name                      | Mandatory / Default | Description                                                        |
-|:-------------------------:|:-------------------:|:------------------------------------------------------------------:|
-| `admins`                  | `[]`                | list of admin users, see [User configuration](#User configuration) |
-| `locales`                 | `[en_US.UTF-8]`       | list of locales to install                                         |
-| `keyboard_layout`         | `us,de`             | TTY keyboard layout                                                |
-| `default_language`        | `en_US.UTF-8`       | default language                                                   |
-| `apt_repository_versions` | `[main, universe]`  | List of apt repository versions to enable                          |
+|            Name           | Mandatory / Default | Description                                                        |
+|:-------------------------:|:-------------------:|--------------------------------------------------------------------|
+|          `admins`         |         `[]`        | list of admin users, see [User configuration](#User configuration) |
+|         `locales`         |   `[en_US.UTF-8]`   | list of locales to install                                         |
+|     `keyboard_layout`     |       `us,de`       | TTY keyboard layout                                                |
+|     `default_language`    |    `en_US.UTF-8`    | default language                                                   |
+| `apt_repository_versions` |  `[main, universe]` | List of apt repository versions to enable                          |
```

Comment:
Can you please stop indenting all tables? This will just be reverted if somebody else changes something at this table and it clutters the diff